app/views/emails/_editForm.html.erb

```diff
@@ -0,0 +1,23 @@
+<body class="title">
+  <%= form_with(model: @email, local: true) do |f| %>
+    <% if @email.errors.any? %>
+      <div id="error_explanation">
+        <h2><%= pluralize(@email.errors.count, "error") %> prohibited this email from being saved:</h2>
+
+        <ul>
+        <% @email.errors.full_messages.each do |message| %>
+          <li><%= message %></li>
+        <% end %>
+        </ul>
+      </div>
+    <% end %>
+
+    <%= f.text_field :subject, class: 'subject', :required => true, :placeholder => 'Subject'%>
+    <br>
+    <%= f.text_field :description, class: 'message', :required => true, :placeholder => 'Message'%>
+    <br>
+    <%= f.date_select :sendDate %>
+    <br>
+    <%= f.submit "Save Edited Email"%>
+  <% end %>
+</body>
```

Comment:
What if you want to change what groups an email's being sent to?

Reply:
Fixed!